crates/holochain/CHANGELOG.md

```diff
@@ -7,14 +7,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-## 0.5.0-dev.4
```

Comment:
This is a really unclear diff, I noticed that the changelog entry for unstable-functions had ended up under dev.4 but should be unreleased, so I moved it back up